.buildkite/pipeline.yml

```
@@ -35,104 +42,122 @@ steps:
+            download: "features/fixtures/app/build/ios/ipa/app-{{matrix}}.ipa"
+            upload:
+              - "maze_output/maze_output.zip"
+              - "maze_output/failed/**/*"
+          docker-compose#v4.7.0:
+            pull: "maze-runner"
+            run: "maze-runner"
+            service-ports: true
+            command:
+              - "--app=/app/features/fixtures/app/build/ios/ipa/app-{{matrix}}.ipa"
+              - "--farm=bb"
+              - "--device=IOS_14|IOS_15|IOS_16"
+              - "--no-tunnel"
+              - "--aws-public-ip"
+              - "--fail-fast"
+          test-collector#v1.10.2:
+            files: "reports/TEST-*.xml"
+            format: "junit"
+            branch: "^main|next$$"
+        concurrency: 25
+        concurrency_group: "bitbar"
+        concurrency_method: "eager"
+        retry:
+          automatic:
+            - exit_status: 103  # Appium session failed
+              limit: 2
 
-  - label: ":browserstack: Android 12 end-to-end tests 3.10.0"
-    depends_on: "android-fixture-3-10-0"
-    timeout_in_minutes: 20
-    agents:
-      queue: "opensource"
-    plugins:
-      artifacts#v1.5.0:
-        download: "features/fixtures/app/build/app/outputs/flutter-apk/app-release-3.10.0.apk"
-        upload:
-          - "maze_output/maze_output.zip"
-          - "maze_output/failed/**/*"
-      docker-compose#v4.7.0:
-        pull: "maze-runner"
-        run: "maze-runner"
-        service-ports: true
-        command:
-          - "--app=/app/features/fixtures/app/build/app/outputs/flutter-apk/app-release-3.10.0.apk"
-          - "--farm=bs"
-          - "--device=ANDROID_12"
-          - "--appium-version=1.22.0"
-          - "--fail-fast"
-      test-collector#v1.10.2:
-        files: "reports/TEST-*.xml"
-        format: "junit"
-        branch: "^main|next$$"
-    concurrency: 5
-    concurrency_group: "browserstack-app"
-    concurrency_method: "eager"
-    retry:
-      automatic:
-        - exit_status: 103  # Appium session failed
-          limit: 2
+      - label: ":browserstack: Flutter {{matrix}} Android 12 end-to-end tests"
+        depends_on: "android-fixture"
+        timeout_in_minutes: 20
+        matrix:
+#          - "3.38.3" Skipped: Waiting on PLAT-15326
+          - "3.24.4"
+          - "3.19.0"
+          - "3.10.0"
+        agents:
+          queue: "opensource"
+        plugins:
+          artifacts#v1.5.0:
+            download: "features/fixtures/app/build/app/outputs/flutter-apk/app-release-{{matrix}}.apk"
+            upload:
+              - "maze_output/maze_output.zip"
+              - "maze_output/failed/**/*"
+          docker-compose#v4.7.0:
+            pull: "maze-runner"
+            run: "maze-runner"
+            service-ports: true
+            command:
+              - "--app=/app/features/fixtures/app/build/app/outputs/flutter-apk/app-release-{{matrix}}.apk"
+              - "--farm=bs"
+              - "--device=ANDROID_12"
```

It's not worth going around the loop again on this PR, but we could open this up to give it more device availability.  Just update the step label to remove "12" if/when we do it.
```suggestion
              - "--device=ANDROID_12|ANDROID_13|ANDROID_14|ANDROID_15"
```

---

My plan was to do that and migrate it to BB as part of the fixture rebuild.

---

I didn't even notice it was on BS!   My comment isn't valid.